### PR TITLE
Add jrubyscripting to automation add-ons

### PR DESCRIPTION
**Pull request by jimtng**
For the newly merged jrubyscripting automation add-on. There doesn't seem to be any specific rule for ordering the items.

### CODE REVIEW

**Comment by ghys** [comment]
Actually now I would start to feature the major script languages add-ons (JS, Jython, JRuby, Groovy) since there are now 4 of them, and "unfeature" the more esoteric/niche ones like pidcontroller (no offense!).
This would be done by removing the "all: true" parameter.
I'd still appreciate if there was a JRuby logo in the docs repo before I merge this.

**Comment by jimtng** [comment]
> I'd still appreciate if there was a JRuby logo in the docs repo before I merge this.

Hi Yannick, thanks for your work. The jruby logo can be found here https://github.com/jruby/collateral
I know you've explained about the legalities, but I'm still paranoid about submitting a PR related to logos, so if you wouldn't mind doing it, it would be great thanks.

> "unfeature" the more esoteric/niche ones like pidcontroller (no offense!).

There seem to be two different sub-categories within "automation", i.e. general scripting languages, vs specialised automation.

**Comment by Confectrician** [comment]
@ghys the logo should be available now.

Anything we need to wait for or could this pr get merged?

**Review by ghys — APPROVED** [review]
nope :)

**Pull request merged**
